pyswarms/backend/handlers.py

```diff
@@ -600,8 +600,8 @@ def __call__(self, start_opts, **kwargs):
             if not self.strategy:
                 return start_opts
             return_opts = copy(start_opts)
-            for key in start_opts.keys():
-                if key in self.strategy.keys():
+            for key in start_opts:
```

Comment:
Maybe there's a more specific variable name than `key`? If we're iterating across a parameter or a config I think we can use something like`cfg` etc.

Reply:
Sorry for the late reply. How about `opt`/`options` since we're iterating over the keys in `start_opts`, `end_opts`, etc.?

Reply:
That works! Any is fine :)